python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_base_group_chat_manager.py

```diff
@@ -266,6 +293,12 @@ async def select_speaker(self, thread: List[BaseAgentEvent | BaseChatMessage]) -
         topic type of the selected speaker."""
         ...
 
+    async def select_speakers(self, thread: List[BaseAgentEvent | BaseChatMessage]) -> List[str]:
```

Comment:
We should just rename `select_speaker` to `select_speakers`, and update the signature so it always returns `Sequence[str]`. 

It is not used publically we can still change it.